utilities/ondc-crypto-sdk-nodejs/README.md

```diff
@@ -0,0 +1,34 @@
+A signing and verification utility that is necessary for creation and verification of authentication signature between network participants of ONDC.
+
+## Signing Authorisation Header
+
+For signing the verification header, you can use the `createAuthorizationHeader` method.
+
+```javascript
+import { createAuthorizationHeader } from "ondc-crypto-sdk-nodejs"
+
+const header = await createAuthorizationHeader({
+      message: { context: {...}, message: {...} },
+      privateKey: privateKey,
+      bapId: "...", // Subscriber ID that you get after registering to ONDC Network
```

Comment:
subscriberId attribute should be used instead of single entity specific (bapId)

Reply:
changed to SubscriberId